conans/client/command.py

```diff
@@ -1941,6 +1941,27 @@ def lock(self, *args):
                                     lockfile=args.lockfile,
                                     lockfile_out=args.lockfile_out)
 
+    def version(self, *args):
+        """ Show Conan version
+        """
+        parser = argparse.ArgumentParser(description=self.version.__doc__,
+                                         prog="conan version",
+                                         formatter_class=SmartFormatter)
+        parser.add_argument("-j", "--json", default=None, action=OnceArgument,
+                            help='Path to a json file where the version information will be '
+                                 'written')
+
+        args = parser.parse_args(*args)
+        if args.json:
+            json_content = json.dumps({"version": client_version})
```

Comment:
`self._outputer.json_output` requires an api cache, but we don't invoke any api method here.